week1/vue_1/src/components/AddingInput.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <div class="inputArea">
+        <input 
+            type="text"
+            v-model="title"
+            v-on:keyup.enter="addTodoItem"
```

Comment:
```suggestion
            @keyup.enter="addTodoItem"
```
`v-on`은 이렇게 생략할 수 있어요!